test/multiverse/suites/dynamodb/Envfile

```diff
@@ -0,0 +1,10 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+instrumentation_methods :chain, :prepend
+
+gemfile <<~RB
+  gem 'aws-sdk-dynamodb'
+  gem 'nokogiri'
```

Comment:
What is nokogiri used for?

Reply:
Before I added nokogiri, the aws sdk failed to install and said "you need to install one of these gems too", and nokogiri was on the list of options, so I just added that one so the aws sdk would work. I'm not using it directly.